FastBank.Services/CustomerService/CustomerService.cs

```diff
@@ -48,12 +49,26 @@ public List<string> ValidatеCustomer(Customer customer)
             CustomerExist(customer, validationErrors);
 
             //TODO validate password 
-            //TODO validate email
+            ValidateEmail(customer.Email, validationErrors);
             CustomerAgeIsValid(customer, validationErrors);
             //TODO validate role
             return validationErrors;
         }
 
+        public List<string> ValidateEmail(string email, List<string> validationErrors)
+        {
+            var regEmailPattern = new Regex("^[\\w-\\.]+@([\\w-]+\\.)+[\\w-]{2,4}$");
+            if (regEmailPattern.IsMatch(email))
+            {
+                return validationErrors;
+            }
+            else
+            {
+                validationErrors.Add("You entered wrong email");
+                return validationErrors;
+            }
+        }
```

Comment:
In this method you need only to check if it doesn't match and add error in the validation errors list. Please rewrite the method.